darwin/index.js

```diff
@@ -88,7 +92,7 @@ const start = () => {
     }, 2000);
 
     if (isDev) {
-      player.loadURL('http://127.0.0.1:3001');
+      player.loadURL('http://lvh.me:3001');
```

Comment:
Why the switch?

Reply:
many yt videos are blocked when request is coming from 127.0.0.1

Reply:
The Youtube API seems to not work properly if you load the player from `127.0.0.1`, some videos were skipped without giving any errors. `lvh.me` is a dummy website the redirects to `127.0.0.1`, this way Youtube doesn't complain :)